### DISCUSSION TITLE
[@types/d3-selection] transition is missing

### DISCUSSION THREAD

**Discussion by simPod** [DefinitelyTyped/DefinitelyTyped#54131] [Issues with a @types package]
It is possible to use `transition()` on Selection like this

```ts
const fade = (opacity: number) => (e: MouseEvent, g: d3Chord.ChordGroup) => svg.selectAll('g path')
  .transition()
  .style('opacity', opacity);
```

however, v3 of types says there's no such function

**Comment by Fil** [comment]
I don't know how to help you, but note that transitions live in [d3-transition](https://github.com/d3/d3-transition), not d3-selection.

**Reply by simPod** [reply]
Thank you for response. I'm kind of lost in d3 types as well though `selectAll()` returns `Selection` type so I thought `transition()` would exist there. But you're right, just found https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/d3-transition/index.d.ts#L44 I'll have to investigate further.
